src/main/kotlin/cacophony/controlflow/functions/FunctionHandlerImpl.kt

```diff
@@ -18,7 +19,9 @@ class FunctionHandlerImpl(
     private val ancestorFunctionHandlers: List<FunctionHandler>,
     callConvention: CallConvention,
     private val variablesMap: VariablesMap,
+    // TODO (?): closure analysis is not really used in the FunctionHandler, as it does not need to generate a closure
```

Comment:
You can add this to the list of closure problems 😄 https://github.com/mhorod/cacophony/issues/305